### ISSUE TITLE
GUI option/feature to continue training from a saved snapshot

### ISSUE THREAD

**Issue by hkosiara** [DeepLabCut/DeepLabCut#2757]
I have been using deeplabcut for about 3 weeks, and after asking around on forums, I was told that DLC GUI does not have a way to continue training the network from a saved snapshot. That is, I have a snapshot of a network at 650 epochs, and I would like to train it for an additional 100 epochs, to 750. There is a way to do this in the terminal, but I use the GUI, and it seems like it's an important feature to have in both interfaces. There used to be a way to do it in the pose.cfg file with tensorflow, but I tried something similar with the new pytorch version to no avail. I think there should be an option to load a snapshot file path and continue training after that snapshot so that users can improve upon network training rather than having to start from 0 epochs each time more training is desired.

**Comment by n-poulsen** [comment]
Hi @hkosiara! This is a great feature request, and I'll get to work on it now

**Comment by austin2118ace** [comment]
@n-poulsen 
> Hi @hkosiara! This is a great feature request, and I'll get to work on it now

Is this feature available in the CLI version of DLC 3.0, or do we have to start from the pretrained weights each time? I looked through the config files as @hkosiara did and also didn't notice anywhere that I could load a snapshot.

**Comment by n-poulsen** [comment]
@austin2118ace this is already available through the CLI for `DeepLabCut 3.0`. When calling [`train_network`](https://github.com/DeepLabCut/DeepLabCut/blob/17d72054a0382ecbabb6ba268da54539e5296bc9/deeplabcut/compat.py#L60) for PyTorch models, any additional parameters for the PyTorch implementation of [`train_network`](https://github.com/DeepLabCut/DeepLabCut/blob/17d72054a0382ecbabb6ba268da54539e5296bc9/deeplabcut/pose_estimation_pytorch/apis/train.py#L194) (found in `deeplabcut/pose_estimation_pytorch/apis/train.py`) can be passed through the [`**torch_kwargs`]((https://github.com/DeepLabCut/DeepLabCut/blob/17d72054a0382ecbabb6ba268da54539e5296bc9/deeplabcut/compat.py#L76)). So currently you can call:

```python
import deeplabcut

deeplabcut.train_network(
    "/path/to/my/dlc-project",
    shuffle=12,
    snapshot_path="/path/to/resume/training/from/snapshot-100.pt",
)
```

**Comment by austin2118ace** [comment]
> @austin2118ace this is already available through the CLI for `DeepLabCut 3.0`. When calling [`train_network`](https://github.com/DeepLabCut/DeepLabCut/blob/17d72054a0382ecbabb6ba268da54539e5296bc9/deeplabcut/compat.py#L60) for PyTorch models, any additional parameters for the PyTorch implementation of [`train_network`](https://github.com/DeepLabCut/DeepLabCut/blob/17d72054a0382ecbabb6ba268da54539e5296bc9/deeplabcut/pose_estimation_pytorch/apis/train.py#L194) (found in `deeplabcut/pose_estimation_pytorch/apis/train.py`) can be passed through the `**torch_kwargs`. So currently you can call:
> 
> ```python
> import deeplabcut
> 
> deeplabcut.train_network(
>     "/path/to/my/dlc-project",
>     shuffle=12,
>     snapshot_path="/path/to/resume/training/from/snapshot-100.pt",
> )
> ```

Well I'll be damned, I promise I searched for that! Thanks for pointing it out to us :)
